Gemfile

```diff
@@ -27,4 +27,5 @@ group :test do
   gem 'capybara'
   gem 'webmock', '~> 1.18.0', :require => false
   gem 'govuk-content-schema-test-helpers', '1.1.0'
+  gem 'pry'
```

Comment:
Add `pry-byebug` instead for even more wealth, greatness and the ability to advance up and down stack frames.